src/containers/logout.jsx

```diff
@@ -0,0 +1,14 @@
+// @flow
+import { Component } from "react";
```

Comment:
logout as container? Why do you need it?

Reply:
It is the exit processing page and Redirect to the main page. This container will transmit an exit signal.

Reply:
We'll do all this logic in 'logout' saga, delete this file.